plugin/pkg/admission/cloudprovider/node/admission.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package node
+
+import (
+	"encoding/json"
+	"io"
+
+	"k8s.io/apiserver/pkg/admission"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/v1"
+	utiltaints "k8s.io/kubernetes/pkg/util/taints"
+
+	"github.com/golang/glog"
+)
+
+const (
+	CloudProviderTaint = "CloudProviderProcessing=Required:NoSchedule"
```

Comment:
This is boolean `Required/Done`?
I like the wording in `CloudProviderInitialized=true/false` better. I can change it to that.

*Naming is hard*